diff --git a/dist/utils/spinnies.js b/dist/utils/spinnies.js
index 98b1ade7a12532161e3be7acb4b848b7b3830986..fade9ce0ce8fdc27956ff989ddc1cefacf85d22c 100644
--- a/dist/utils/spinnies.js
+++ b/dist/utils/spinnies.js
@@ -10,6 +10,9 @@ function getSpinnies(options) {
     if (!spinnies) {
         spinnies = new spinnies_1.default(options);
     }
+    spinnies.fail = () => null;
+    spinnies.succeed = () => null;
+    spinnies.add = () => null;
     return spinnies;
 }
 exports.getSpinnies = getSpinnies;
